projects/1.20/assets/born-in-chaos/born_in_chaos_v1/lang/zh_cn.json

```
@@ -437,45 +545,100 @@
+  "item.born_in_chaos_v1.coffee_candy": "咖啡糖",
   "item.born_in_chaos_v1.controlled_baby_skeleton_spawn_egg": "受控的小骷髅刷怪蛋",
   "item.born_in_chaos_v1.controlled_spiritual_assistant_spawn_egg": "受控的随从小骷髅刷怪蛋",
   "item.born_in_chaos_v1.corpse_fish_spawn_egg": "腐尸鱼刷怪蛋",
   "item.born_in_chaos_v1.corpse_fly_spawn_egg": "腐尸苍蝇刷怪蛋",
   "item.born_in_chaos_v1.corpse_maggot": "尸蛆",
+  "item.born_in_chaos_v1.creepy_cookies_with_milk.description_0": "§9移除所有效果",
+  "item.born_in_chaos_v1.creepy_cookies_with_milk.description_1": "§9淘气值-3",
+  "item.born_in_chaos_v1.creepy_cookies_with_milk": "悚异的曲奇牛奶",
+  "item.born_in_chaos_v1.creepy_gift.description_0": "§8右击以打开",
+  "item.born_in_chaos_v1.creepy_gift": "悚异的礼物",
+  "item.born_in_chaos_v1.damned_demomans_hat_helmet.description_0": "§c—这顶帽子无法摘下",
+  "item.born_in_chaos_v1.damned_demomans_hat_helmet.description_1": "§c—生命垂危之时，你将会爆炸",
   "item.born_in_chaos_v1.damned_demomans_hat_helmet": "§6诅咒爆破帽",
+  "item.born_in_chaos_v1.dark_atrium.description_0": "§8使用时，获得§9暗黑监护（60:00）§8效果，可使你免于一死",
+  "item.born_in_chaos_v1.dark_atrium": "§c暗黑心房",
   "item.born_in_chaos_v1.dark_charge": "暗影弹",
   "item.born_in_chaos_v1.dark_metal_armor_boots": "§6暗金属靴子",
   "item.born_in_chaos_v1.dark_metal_armor_chestplate": "§6暗金属胸甲",
+  "item.born_in_chaos_v1.dark_metal_armor_helmet.description_0": "§7全套装备时：",
+  "item.born_in_chaos_v1.dark_metal_armor_helmet.description_1": "§8—生命垂危时，你将进入暴怒状态",
+  "item.born_in_chaos_v1.dark_metal_armor_helmet.description_2": "§8—凋零抗性",
   "item.born_in_chaos_v1.dark_metal_armor_helmet": "§6暗金属头盔",
   "item.born_in_chaos_v1.dark_metal_armor_leggings": "§6暗金属护腿",
   "item.born_in_chaos_v1.dark_metal_ingot": "暗金属锭",
+  "item.born_in_chaos_v1.dark_metal_nugget": "暗金属粒",
+  "item.born_in_chaos_v1.dark_ritual_dagger.description_0": "§7—右击一个动物或侍从以献祭它，给予你生命恢复和力量效果",
+  "item.born_in_chaos_v1.dark_ritual_dagger.description_1": "§8—扩展并增强献祭的效果",
   "item.born_in_chaos_v1.dark_ritual_dagger": "§6暗黑仪式",
   "item.born_in_chaos_v1.dark_rod": "暗影棒",
+  "item.born_in_chaos_v1.dark_upgrade.description_0": "§7暗金属升级",
+  "item.born_in_chaos_v1.dark_upgrade.description_1": "",
+  "item.born_in_chaos_v1.dark_upgrade.description_2": "§7可应用于：",
+  "item.born_in_chaos_v1.dark_upgrade.description_3": " §9下界合金装备",
+  "item.born_in_chaos_v1.dark_upgrade.description_4": "§7所需原材料：",
+  "item.born_in_chaos_v1.dark_upgrade.description_5": " §9暗金属装甲板",
   "item.born_in_chaos_v1.dark_upgrade": "锻造模板",
   "item.born_in_chaos_v1.dark_vortex_spawn_egg": "暗影漩涡刷怪蛋",
+  "item.born_in_chaos_v1.darkwarblade.description_0": "§7—攻击具有崩山碎石的威力",
+  "item.born_in_chaos_v1.darkwarblade.description_1": "§c—这柄武器太重了，如果没有力量效果将难以使用",
   "item.born_in_chaos_v1.darkwarblade": "§6暗黑大剑",
-  "item.born_in_chaos_v1.death_totem": "死灵图腾",
+  "item.born_in_chaos_v1.death_totem.description_0": "§7手持时：",
+  "item.born_in_chaos_v1.death_totem.description_1": "§8—消耗你的饥饿值，为你挡下一次致命伤害，并获得力量效果",
+  "item.born_in_chaos_v1.death_totem.description_2": "§7全套梦魇装备时：",
```

full set of Nightmare Armor
指的是盔甲吧
```suggestion
  "item.born_in_chaos_v1.death_totem.description_2": "§7全套梦魇盔甲装备时：",
```